pkg/server/commands/connected_objects.go

```diff
@@ -554,10 +554,6 @@ func (c *ConnectedObjectsCommand) reverseExpandDirect(
 			},
 		}
 
-		if tuple.IsTypedWildcard(foundObject) {
-			targetUserRef = &UserRefTypedWildcard{Type: foundObjectType}
-		}
```

Comment:
this is never hit by our tests. Plus i think it can never happen, since you can't have `user:*` as the `object` field of a tuple 🤔

Reply:
do you mean we should do something like this?

```go
if tuple.IsTypedWildcard(foundObject) {
			panic("did not expect type:* on the object field of a tuple")
		}
```

hmmmmmm

> should these be a panic/assert (0)?

Worry that later on when we relax the model restriction and we forgot to update this

what do you mean by relax model restrictions?

Reply:
I meant that if/when we allow wildcard in TTU later on for model schema 1.1.

Reply:
So for example you say what if in the future this test case  doesn't throw an error?

https://github.com/openfga/openfga/blob/eb8e348ced49a4bb514400ed720b7acabef52e9a/internal/validation/validation_test.go#L426-L468

if/when we change this, i think the code i deleted still won't be run ever

Reply:
I see.